pkgs/development/python-modules/pyglet/default.nix

```diff
@@ -1,26 +1,31 @@
 { lib, stdenv
 , buildPythonPackage
 , fetchPypi
+, unzip
+, isPy3k
 , libGL
 , libGLU
 , xorg
-, future
 , pytest
 , glibc
 , gtk2-x11
 , gdk-pixbuf
 , fontconfig
 , freetype
 , ffmpeg-full
+, openal
+, libpulseaudio
 }:
 
 buildPythonPackage rec {
-  version = "1.4.2";
+  version = "1.5.19";
   pname = "pyglet";
+  disabled = !isPy3k;
```

Comment:
Please use pythonOlder.

Reply:
Done, documentation says <3.6 is not supported.